kube-burner/hcp-density-aks/hcp-density-aks.yml

```diff
@@ -0,0 +1,60 @@
+---
+global:
+  indexerConfig:
+    esServers: [http://elastic-elk.apps.rsevilla-ocp45-ovn.perf-testing.devcluster.openshift.com]
+    insecureSkipVerify: false
+    defaultIndex: kube-burner
+    type: elastic
+  measurements:
+    - name: podLatency
+  clusterHealth: true
+
+jobs:
```

Comment:
config maps & secrets ?

Reply:
Did not find any secrets/configmaps here :(
https://gist.github.com/mukrishn/05964efa55cde274067375aa1bfd2ef8

Reply:
my bad, `get all` didn't capture it. let me update it

Reply:
updated it, there are approxly 110 secrets. keep the same count for configmap too.